packages/x-components/src/x-modules/tagging/service/__tests__/session.service.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { DefaultSessionService } from '../session.service';
+import { MockedStorageService } from './utils/mock-storage.service';
+
+describe('testing session id service', () => {
+  const prefix = 'test';
+  const mockedStorageService = new MockedStorageService(prefix);
```

Comment:
In order to test the SessionService I mocked a custom service in order to spy the calls and the payloads.